boston-housing/index.js

```diff
@@ -47,44 +65,41 @@ export const linearRegressionModel = () => {
  */
 export const multiLayerPerceptronRegressionModel = () => {
   const model = tf.sequential();
-  model.add(tf.layers.dense(
-      {inputShape: [data.numFeatures], units: 50, activation: 'sigmoid'}));
+  model.add(tf.layers.dense({
+    inputShape: [bostonData.numFeatures],
+    units: 50,
+    activation: 'sigmoid'
+  }));
   model.add(tf.layers.dense({units: 50, activation: 'sigmoid'}));
   model.add(tf.layers.dense({units: 1}));
 
   return model;
 };
 
 /**
- * Fetches training and testing data, compiles `model`, trains the model
- * using train data and runs model against test data.
+ * Fetches training and testing data, normalizes, compiles `model`, trains the
```

Comment:
`run` no longer fetches or normalizes data. We can move this part `arraysToTensors`

Reply:
Thanks for catching this!  I should go back through the doc to see if the comments are still correct.